rosie.py

```diff
@@ -26,12 +31,19 @@ def run():
     if len(argv) >= 3:
         target_module = argv[2]
     else:
-        print('A module must be provided.')
-        help()
+        arguments = docopt(__doc__, help=True)
```

Comment:
Why parsing arguments with `docopt` here and not in main? Isn't `docopt` used for both `run` and `test`?

What about parsing arguments else where (`main`?) and calling `run` or `test` when arguments are already parsed?